weblate/checks/format.py

```diff
@@ -340,7 +344,9 @@ def normalize(self, matches: list[str]) -> list[str]:
     def extract_matches(self, string: str) -> list[str]:
         return [self.cleanup_string(x[0]) for x in self.regexp.findall(string)]
 
-    def check_format(self, source: str, target: str, ignore_missing, unit: Unit):
+    def check_format(
```

Comment:
The doc strings has potential to be improved. Not clear what it does. Arguments and return values are unclear.

Because of that I am unable to value the new unit tests using this method.

Reply:
Indeed, the documentation always has potential to get improved. Currently, I'm at least focusing on adding type annotations when touching the code.